fastai/gen_doc/doctest.py

```diff
@@ -11,7 +14,7 @@ def this_tests(testedapi):
         prev_frame = inspect.currentframe().f_back.f_back
         pathfilename, line_number, test_function_name, lines, index = inspect.getframeinfo(prev_frame)
         lineno_parentfunc, parent_func = get_parent_func(line_number, get_lines(pathfilename))
-        list_test = [{'file': basename(pathfilename), 'test': test_function_name , 'line': str(lineno_parentfunc)}]
+        list_test = [{'file': relative_test_path(pathfilename), 'test': test_function_name , 'line': lineno_parentfunc}]
```

Comment:
@Benudek 'file' now includes the 'tests' folder along with the name.

This was easier for me to link the github sourcecode and pytest command. I can change it back if you see any problems.